### DISCUSSION TITLE
Can not set the default model

### DISCUSSION THREAD

**Discussion by miesgre** [open-webui/open-webui#2820] [General]
# Bug Report

## Description

After updating to version 0.2.4 from 0.1.125, the default model has stopped working
In my environment variables I have: DEFAULT_MODELS=llama3
but it does not work. How can I select a default model for users?

**Expected Behavior:**
model llama3 to be the default model.

**Actual Behavior:**
There is no default model

## Environment

- **Open WebUI Version:** [0.2.4]

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.

**Comment by justinh-rahb** [comment]
Global default model has been decoupled from the admin's default setting and is adjusted from this menu now.

![Screenshot_20240604-080633.png](https://github.com/open-webui/open-webui/assets/52832301/9c676cb5-91c4-47c6-aded-00975e26a91b)

**Reply by d1egoaz** [reply]
I wonder if this feature was replaced, I can't find this menu

**Comment by miesgre** [comment]
> Global default model has been decoupled from the admin's default setting and is adjusted from this menu now.

The default model appears in that menu, but when I open a new chat it is not selected

![Captura de pantalla 2024-06-04 a las 14 02 07](https://github.com/open-webui/open-webui/assets/2735202/06987fdc-a19a-4994-b51b-d996763da541)

**Comment by justinh-rahb** [comment]
> > Global default model has been decoupled from the admin's default setting and is adjusted from this menu now.
> 
> The default model appears in that menu, but when I open a new chat it is not selected
> 
> ![Captura de pantalla 2024-06-04 a las 14 02 07](https://private-user-images.githubusercontent.com/2735202/336457326-06987fdc-a19a-4994-b51b-d996763da541.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MTM3NzksIm5iZiI6MTcxNzUxMzQ3OSwicGF0aCI6Ii8yNzM1MjAyLzMzNjQ1NzMyNi0wNjk4N2ZkYy1hMTlhLTQ5OTQtYjUxYi1kOTk2NzYzZGE1NDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMTUwNDM5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTgwN2Y5MmJiMTFhZmNmZjVlNzExZDQ0ODZjNGI4MjE3NDg4NTUyM2JjOTViZTgzMWViMzRkZGY4ZTJmMGQ3NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lXX1Dr8HeOgCfwGuKRfZTNa_kpqBzeG07sH5HTwNfik)

Most likely you previously had a user default setting with a model that no longer exists (or which was renamed). You can just set a new default, or remove all site data from your browser for your WebUI instance and it should reset:

![Screenshot 2024-06-04 at 11 06 29 AM](https://github.com/open-webui/open-webui/assets/52832301/2e388b0f-ce67-4968-a459-247820ea608c)
![Screenshot 2024-06-04 at 11 06 44 AM](https://github.com/open-webui/open-webui/assets/52832301/72ddd650-8313-4487-9e36-315e28a6c6fe)
